tools/nl/svindex_differ/template.html

```diff
@@ -40,15 +41,59 @@ <h1>Queries with Diffs</h1>
     <pre>Base: {{base_file}}</pre>
     {% if diffs %}
       <ul>
-      {% for query, base, test in diffs %}
+      {% for query, diff_table_rows in diffs %}
         <li><a href="#{{ loop.index }}">{{ loop.index }}. {{query}}</a></li>
       {% endfor %}
       </ul>
 
-      {% for query, base, test in diffs %}
-        <hr>
+      {% for query, diff_table_rows in diffs %}
         <h2 id="{{ loop.index }}">{{ loop.index }}: {{query}}</h2>
-        {{ base|diff_table(test) }}
+        <button class="test">TEST</button>
```

Comment:
What's this for?

Reply:
this was testing code that I forgot to delete